src/chainstate/burn/db/sortdb.rs

```diff
@@ -738,7 +793,34 @@ const SORTITION_DB_SCHEMA_5: &'static [&'static str] = &[r#"
         amount TEXT NOT NULL,
 
         PRIMARY KEY(txid)
-    );"#];
+    );"#,
+    r#"
+    CREATE TABLE peg_out_requests (
+        txid TEXT NOT NULL,
+        vtxindex INTEGER NOT NULL,
+        block_height INTEGER NOT NULL,
+        burn_header_hash TEXT NOT NULL,
+
+        amount TEXT NOT NULL,
+        recipient TEXT NOT NULL,
+        signature TEXT NOT NULL,
+
+        PRIMARY KEY(txid)
```

Comment:
As with peg-ins, you're going to need to use a primary key that includes both the `txid` and the sortition ID in which it was processed (per #3527).  This applies to both of these two new tables.

Reply:
Fixed this while rebasing on the updated peg-in branch.